docs/ise_compatibility.md

```diff
@@ -0,0 +1,76 @@
+# ISE Compatibility
+
+While the PowerShell extension for VSCode does not seek
+100% compatibility/reproduction of features in the PowerShell ISE,
+there are features in place to make the VSCode experience more natural
+for users of the ISE.
+
+This document tries to list settings you can configure in VSCode
+to make the user experience a bit more familiar compared to the ISE.
+
+## Key bindings
```

Comment:
ehem... ever heard of <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>J</kbd>... it's been in since 1.10 :smile:

It shows all the snippets you have available.

Reply:
Might be worth mentioning <kbd>F8</kbd> here too.

Reply:
What's the <kbd>F8</kbd> equivalent in the ISE?

Reply:
It's `F8` 😁